takwimu/templates/takwimu/_includes/homepage/hero_detail.html

```diff
@@ -3,13 +3,14 @@
 <div class="carousel-item p-5 {% if forloop.first %}active{% endif %}">
   <div class="">
     <img src="{% static 'img/flags/'|add:country|add:'.svg' %}" class="float-right"/>
-    <p class="text-uppercase" style="font-weight: 600;">New Country</p>
+    <p class="text-uppercase" style="font-weight: 400; font-size:13px;">New Country</p>
     <h2 style="font-weight: 600;">{{ country|title }} country profile now available</h2>
-    <p>Explore human development data and analysis for {{ country|title }}.</p>
+    <p class="pt-2">Explore human development data and analysis for {{ country|title }}.</p>
     <p>
-      <a href="/profiles/{{ country }}" class="btn btn-info px-4 text-uppercase" style="font-size: 80%;">
-        View Country Profile <i class="fa fa-arrow-right ml-2"></i>
+      <a href="/profiles/{{ country }}" class="btn btn-info px-4 text-uppercase" style="font-size: 80%; height: 44px;">
+        View Country Profile <i class="fas fa-long-arrow-alt-right fa-2x ml-2 pt-2"></i>
```

Comment:
1. Let's use the icon shared by Sean on Zeplin
2. The `pt-2` is causing design inconsistency. Please remove.
3. Let's use `max-height` or `min-height` to remain responsive.